pom.xml

```diff
@@ -2613,6 +2613,7 @@
               <jvmArg>-Xss128m</jvmArg>
               <jvmArg>-Xms2048m</jvmArg>
               <jvmArg>-Xmx2048m</jvmArg>
+              <jvmArg>-XX:MaxMetaspaceSize=1g</jvmArg>
```

Comment:
Shall we make it `2g` here?

Reply:
Although I didn't do that because `-Xmx2048m` in this context, but I can set it 2g.
Or, does it make sense to increase `-Xmx4g` together?

Reply:
Metaspace should be outside the heap, so I think it's OK to set 2g here. You're welcome to increase the heap too if you like

Reply:
Oh, right.

Reply:
Yep, then, let me increase `-Xmx` together. Thanks, @srowen .